### ISSUE TITLE
Better image support

### ISSUE THREAD

**Issue by josevalim** [livebook-dev/kino#155]
There are some ideas flying around:

1. Support Nx.Tensor in Kino.Image by using a custom content type that is loaded into a canvas
2. Support StbImage in Kino.Image
3. Have a canvas input
4. Have a specific image tensor upload, where the image is loaded into a canvas to be converted into pixels (and potentially resized)

**Comment by cocoa-xu** [comment]
Hi @josevalim, I'd like to add support for camera input if no one is working on it. :)

**Comment by josevalim** [comment]
Yes, please go ahead. Let’s start with camera image first, so there is an explicit click button to take a picture?

**Comment by cocoa-xu** [comment]
Yeah, I'm thinking something like this:

<img width="445" alt="Screenshot 2022-12-18 at 09 27 32" src="https://user-images.githubusercontent.com/89497197/208291151-e187917c-5ea4-4ad1-91c6-706832384d2c.png">

And, of course, we can probably also make it optional here, something like `Kino.Input.image("Image", camera: false)`.

Linked PR: https://github.com/livebook-dev/livebook/pull/1595

**Comment by josevalim** [comment]
Closing for now! Please ping if there is interest in the canvas mode.
